src/compositemark/errorbar.ts

```diff
@@ -69,7 +69,10 @@ export interface ErrorExtraEncoding<F extends Field> {
   yError2?: SecondaryFieldDef<F> | ValueDef<number>;
 }
 
-export type ErrorEncoding<F extends Field> = Pick<Encoding<F>, PositionChannel | 'color' | 'detail' | 'opacity'> &
```

Comment:
@kanitw @domoritz I'm not sure why `size` is not in the `ErrorEncoding`. Is it not supposed to be in the encoding channel of error bar?

Reply:
Isn't `ErrorEncoding` shared between ErrorBar and ErrorBand?